.github/actions/install-metal-deps/dependencies.json

```diff
@@ -5,7 +5,6 @@
     "python3.8-venv=3.8.10-0ubuntu1~20.04.9",
     "libgoogle-glog-dev=0.4.0-1build1",
     "libyaml-cpp-dev=0.6.2-4ubuntu1",
-    "libboost-all-dev=1.71.0.0ubuntu2",
```

Comment:
Will a simple uninstall work for the current hosts on cloud?

Do we need to go through `lib`s and `include`s etc?

Reply:
Would pkg manager install not remove libs and includes?

Reply:
You never know - I think we could do a quick manual check on a VM and see if anything lingers

Just to ensure that devs don't see anything weird locally

Reply:
Locally on my machine, running `sudo apt-get remove --auto-remove libboost-all-dev` seems to remove `include` and `lib` for boost too.

Reply:
OK beautiful - gonna blast there everywhere once this is in
We should probably make an issue on MIW